blog/migrations/0001_initial.py

```diff
@@ -1,3 +1,4 @@
+"""Initial migrations."""
```

Comment:
you were supposed to not touch migrations as they are autogenerated.

Reply:
@webknjaz Do I understand correctly? I need to add migrations to `.gitignore`? Right?

Reply:
no. you can exclude it in pre-commit tool